src/qibo/hamiltonians/hamiltonians.py

```diff
@@ -860,19 +724,3 @@ def circuit(self, dt, accelerators=None):
         )
 
         return circuit
-
-
-class TrotterHamiltonian:
```

Comment:
Note that there are a few remaining references to this object, for example here https://github.com/qiboteam/qibo/blob/312efb9d6934e85cfdaafcdb7027cdfe46bf87c4/src/qibo/solvers.py#L47 and some other places in the docs (easily found using search).